rpc/js/src/BaseChannel.ts

```diff
@@ -1,13 +1,13 @@
-import { ProtobufMessage } from "@improbable-eng/grpc-web/dist/typings/message";
+import type { ProtobufMessage } from "@improbable-eng/grpc-web/dist/typings/message";
 
 export class BaseChannel {
-	public readonly ready: Promise<any>;
+	public readonly ready: Promise<unknown>;
 
 	private readonly peerConn: RTCPeerConnection;
 	private readonly dataChannel: RTCDataChannel;
-	private pResolve: (value: any) => void;
+	private pResolve: ((value: unknown) => void) | undefined;
```

Comment:
pResolve is not initialized in the constructor, so must be marked as possibly undefined.